features/orabos/file.include/usr/lib/dracut/modules.d/99ensure-hugepages/ensure-hugepages.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
why you are calculating the hugespage size and do not enforce the "right" size vie kernelcmdline?

Reply:
See PR #12